.githooks/pre-commit

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+PROJECT_ROOT=$(dirname `dirname $0`)
+
+set -Euexo pipefail
+npx webpack --env prod
+
+git add $PROJECT_ROOT/assets/js/primer_spec_plugin.js
```

Comment:
It's not clear if these are minified/uglified.  I believe that they should be.  If so, consider using the common `X.min.js`.

Reply:
While it's true that they're uglified and minified in production, that is not the case in development. That said, I guess it's not a big deal to call them "minified"  — that's something that can be fixed later, if necessary.